srcpkgs/volk/template

```diff
@@ -3,16 +3,22 @@ pkgname=volk
 version=1.4
 revision=4
 build_style=cmake
-nocross="not complete for arm"
+hostmakedepends="pkgconf python-cheetah git python-Mako
+ python-six"
+makedepends="boost-devel python-cheetah python-devel
+ python-Mako python-six"
 short_desc="Vector-Optimized Library of Kernels"
 maintainer="Andrew Benson <abenson+void@gmail.com>"
-hostmakedepends="pkg-config python-cheetah git"
-makedepends="boost-devel python-cheetah python-devel python-Mako python-six"
 license="GPL-3.0-or-later"
 homepage="http://libvolk.org/"
 distfiles="https://github.com/gnuradio/volk/archive/v${version}.tar.gz"
 checksum=32131ba17846850c07270bc5897dd2de7130ec02ca029875a36966335120e7bf
 
+case "$XBPS_TARGET_MACHINE" in
+	armv6l-musl)
+		configure_args="-Dhave_mfpu_neon=0";;
```

Comment:
why just the musl variant and not glibc?

Reply:
Fixed this, was a bit of overlook/thought